gulpfile.js

```diff
@@ -72,22 +76,27 @@ gulp.task('scss', () =>
         .pipe(gulp.dest('./dist/dev/assets/css'))
         .pipe(cleanCSS())
         .pipe(rename({suffix: '.min'}))
-        .pipe(gulp.dest('./dist/prod/assets/css'))
-);
+        .pipe(gulp.dest('./dist/prod/assets/css'));
 
-gulp.task('js', ()  =>
+    cb();
```

Comment:
Why do you use callbacks everywhere now? Does returning the gulp pipes no longer work?

Reply:
Actually I just read https://gulpjs.com/docs/en/getting-started/creating-tasks but as of https://gulpjs.com/docs/en/getting-started/async-completion returning streams still work. I removed the callbacks again. My bad.